client/lib/coords.js

```diff
@@ -0,0 +1,19 @@
+export default class Coords {
+  constructor() {
+    Object.setPrototypeOf(this, Array.prototype);
+    for (let i = 10; i < 90; i += 10) {
+      for (let j = 1; j < 9; j++) {
+        this.push(i + j);
+      }
+    }
+  }
+
+  isCoord(number) {
+    for (const coord of this) {
+      if (number === coord) {
+        return true;
+      }
+    }
+    return false;
+  }
+}
```

Comment:
`Object.setPrototypeOf()` is not meant for production code. Is there a reason not to extend `Array`?

```js
class Coords extends Array {
  constructor(...args) {
    super(...args);
    // loop and push
  }

  isCoord(number) {}

}
```